config/config.go

```diff
@@ -239,6 +240,11 @@ func ValidateConfig(c *SnapshotterConfig) error {
 	if len(c.Root) == 0 {
 		return errors.New("empty root directory")
 	}
+	if realPath, err := mount.NormalizePath(c.Root); err != nil {
+		return errors.Wrapf(err, "Root path invalid.")
+	} else {
+		c.Root = realPath
+	}
```

Comment:
Shall we move the path normalization to `ProcessConfigurations()` where we do all the stuff for tweaking configurations?

Reply:
Fixed